policy/modules/system/sysnetwork.if

```diff
@@ -454,6 +454,7 @@ interface(`sysnet_read_config',`
         init_search_pid_dirs($1)
 		allow $1 net_conf_t:dir list_dir_perms;
 		allow $1 net_conf_t:lnk_file read_lnk_file_perms;
+		allow $1 NetworkManager_var_run_t:dir list_dir_perms;
```

Comment:
By the way, this is still wrong because type NetworkManager_var_run_t is external to the sysnetwork module. You would need to call a interface from the NetworkManager module.

In addition to that any reference in sysnetwork module to NetworkManager should be optional. Since one should be able to disable/remote the NetworkManager module (for example if one does not have NetworkManager and instead uses systemd-networkd)